tetris.py

```diff
@@ -293,6 +303,12 @@ def full_line_clear(self):
                         (a, b + 1) for (a, b) in coord_list if b < x_line
                     ]
 
+    def game_over(self):
```

Comment:
Please rename this to `check_game_over` or something. This function can be called even when the game is not actually over yet.

Reply:
Renamed